include/clap/clap.h

```diff
@@ -28,44 +28,37 @@
 #include "entry.h"
 
 #include "factory/plugin-factory.h"
-#include "factory/draft/plugin-invalidation.h"
-#include "factory/draft/preset-discovery.h"
+#include "factory/preset-discovery.h"
 
 #include "plugin.h"
 #include "plugin-features.h"
```

Comment:
```suggestion
```
is it really necessary to include it here? I mean, it's already included via `#include "factory/plugin.h"`.  Also e.g. `id.h` is also not included here